media-video/plasmatube/plasmatube-23.08.49.9999.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+KFMIN=5.81.0
+QTMIN=5.15.0
+inherit ecm gear.kde.org
+
+DESCRIPTION="YouTube video player based on mpv, yt-dlp, and Invidious"
+HOMEPAGE="https://apps.kde.org/plasmatube/"
+
+LICENSE="CC0-1.0 CC-BY-SA-4.0 GPL-2+ GPL-3+"
+SLOT="0"
+KEYWORDS=""
+
+DEPEND="
+	>=dev-qt/qtcore-${QTMIN}:5
+	>=dev-qt/qtdeclarative-${QTMIN}:5
+	>=dev-qt/qtgui-${QTMIN}:5
+	>=dev-qt/qtnetwork-${QTMIN}:5
+	>=dev-qt/qtquickcontrols2-${QTMIN}:5
+	>=dev-qt/qtsvg-${QTMIN}:5
+	>=dev-qt/qtwidgets-${QTMIN}:5
+	>=kde-frameworks/kconfig-${KFMIN}:5
+	>=kde-frameworks/kcoreaddons-${KFMIN}:5
+	>=kde-frameworks/ki18n-${KFMIN}:5
+	>=kde-frameworks/kirigami-${KFMIN}:5
+	media-video/mpv:=[libmpv]
+"
+RDEPEND="${DEPEND}
+	dev-libs/kirigami-addons:5
+	>=dev-qt/qtgraphicaleffects-${QTMIN}:5
+	>=dev-qt/qtmultimedia-${QTMIN}:5[qml]
```

Comment:
Technically not needed, we don't use qtmultimedia ourselves but some parts of kirigami-addons might.

Reply:
"src/contents/ui/videoplayer" imports QtMultimedia

Reply:
You're right, that still applies to 23.08 just not in 9999